colossalai/zero/sharded_optim/sharded_optim_v2.py

```diff
@@ -257,6 +249,22 @@ def zero_grad(self, *args, **kwargs):
     def sync_grad(self):
         pass
 
+    def _register_master_weight(self):
```

Comment:
If it is an independent function, you'd better Deal with master_params has already exist?

Maybe delete attr and set it again?
delattr, setattr

Reply:
This function is used to avoid CodeFactor's issue.
It should not be used in other places.

Reply:
Fine